newswires/client/src/App.tsx

```diff
@@ -110,9 +72,56 @@ export function App() {
 						</p>
 					</EuiToast>
 				)}
-				{status !== 'error' &&
-					state.queryData &&
-					state.queryData.results.length > 0 && (
+				<div
+					css={css`
+						${status === 'offline' && 'padding-top: 84px;'}
+						height: 100%;
+						${(status === 'loading' || status === 'error') && 'display: flex;'}
```

Comment:
```suggestion
						${(status === 'loading' || status === 'error') && 'display: flex; align-items: center;'}
```

iiuc these rules are coupled, so to my mind it makes sense to have them both in the same expression? not blocking though